src/main/java/Duke.java

```diff
@@ -1,10 +1,100 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void inputChecker() {
+        Scanner in = new Scanner(System.in); // Declared a scanner object
+        String line; // Declared a string object to take in user input
+        Task[] list; // Declared a list with string data type
+        list = new Task[100]; // created a list of 100 elements
```

Comment:
If you want to use a fixed-sized array, consider defining by creating a constant of  task size to 100 instead of directly using 100